utils/fasm/src/parameters.cpp

```diff
@@ -4,30 +4,32 @@
 namespace fasm {
 
 void Parameters::AddParameter(const std::string &eblif_parameter, const std::string &fasm_feature) {
-    auto ret = features_.insert(std::make_pair(eblif_parameter, FeatureParameter()));
-
-    VTR_ASSERT(ret.second);
-
-    ret.first->second.feature = fasm_feature;
-    ret.first->second.width = FeatureWidth(fasm_feature);
+    FeatureParameter fp;
+    fp.width = FeatureWidth(fasm_feature);
+    fp.feature = fasm_feature;
+    features_.insert(std::make_pair(eblif_parameter, fp));
 }
 
 std::string Parameters::EmitFasmFeature(const std::string &eblif_parameter, const std::string &value) {
-    auto iter = features_.find(eblif_parameter);
-    if(iter == features_.end()) {
-        return "";
+    std::ostringstream out;
+    int count = 0;
+    auto range = features_.equal_range(eblif_parameter);
```

Comment:
Why not just use a range for loop here?

Reply:
I tried `for(auto i : features_.equal_range(eblif_parameter)` and it didn't work, and I couldn't find anything saying it should work, so this is how it was done on [cppreference.com](https://en.cppreference.com/w/cpp/container/multimap/equal_range#Example).